reference-implementation/run-tests.js

```diff
@@ -2,12 +2,14 @@ const glob = require('glob');
 const path = require('path');
 
 import ReadableStream from './lib/readable-stream';
+import { IsReadableStreamDisturbed } from './lib/readable-stream';
 import WritableStream from './lib/writable-stream';
 import ByteLengthQueuingStrategy from './lib/byte-length-queuing-strategy';
 import CountQueuingStrategy from './lib/count-queuing-strategy';
 import TransformStream from './lib/transform-stream';
 
 global.ReadableStream = ReadableStream;
+global.IsReadableStreamDisturbed = IsReadableStreamDisturbed;
```

Comment:
For example, such tests could never be run in browsers. Maybe they just should be left for other specs' test suites.